packages/webchannel-wrapper/externs/overrides.js

```diff
@@ -62,3 +62,6 @@ goog.net.WebChannel.Options.backgroundChannelTest;
 
 /** @type {boolean|undefined} */
 goog.net.WebChannel.Options.fastHandshake;
+
+/** @type {(!Object<string, boolean|number>|undefined)|undefined} */
```

Comment:
I'm not sure I fully follow this type definition. Can we not remove one of the `undefined`s and remove the parentheses?

See https://cs.corp.google.com/piper///depot/google3/javascript/closure/labs/net/webchannel.js?q=webchannel+internalChannelParams&dr=CSs&l=211

Reply:
Er, good catch!  I had a brain hiccup when I copy/pasted that I misread it to think the existing "|undefined" was inside the Object definition and so I added another one... :-/  Fixed.